survey/forms.py

```diff
@@ -99,7 +100,9 @@ def get_question_initial(self, question, data):
                 elif "[" in answer.body and "]" in answer.body:
                     initial = []
                     unformated_choices = answer.body[1:-1].strip()
-                    for unformated_choice in unformated_choices.split(","):
+                    for unformated_choice in unformated_choices.split(
+                        getattr(settings, "CHOICES_SEPARATOR", ",")
```

Comment:
Would it work by using `settings.CHOICES_SEPARATOR` here and deporting the getattr in `survey/settings.py` ? I don't remember that well but I think every optional settings was done that way.

Reply:
yes, I just added it with case when CHOICES_SEPARATOR could be not set in setting at all

Reply:
done